### PR TITLE
Select wallet and use WalletConnect

### LINKED ISSUE #29: Select wallet and use WalletConnect

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #29 

#### Includes
- New design (look at the issue for the links to zeplin..)
- Move the old connect logic to the new component
- Check the callback passed to the WalletConnectQRCodeModal open method

A few screenshots ...
Screenshot_20191127_110140
Screenshot_20191127_110152
Screenshot_20191127_110215
Screenshot_20191127_110935
Screenshot_20191127_110951

### CODE REVIEW

File: app/src/components/common/modal_connect_wallet/index.tsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+}
+
+export const ModalConnectWallet = (props: Props) => {
+  const context = useWeb3Context()
+
+  if (context.error) {
+    console.error('Error in web3 context', context.error)
+  }
+
+  const [walletSelected, setWalletSelected] = useState('MetaMask')
```
Comment (fvictorio) [inline app/src/components/common/modal_connect_wallet/index.tsx]: Use `Wallet`

File: app/src/components/common/modal_connect_wallet/index.tsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        onClickCloseButton()
+        localStorage.removeItem('CONNECTOR')
+        context.unsetConnector()
+      })
+
+      context.connector.walletConnector.on('connect', () => {
+        WalletConnectQRCodeModal.close()
+      })
+    }
+  }, [context, walletSelected])
```
Comment (fvictorio) [inline app/src/components/common/modal_connect_wallet/index.tsx]: Unnecessary dependency

File: app/src/components/common/button_connect_wallet/index.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  margin-left: 11px;
+`
+
+interface Props {
+  onClick: () => void
+  modalState: boolean
+}
+
+export const ButtonConnectWallet = (props: Props) => {
+  const context = useWeb3Context()
```
[the remark this answers is not in the document]
Comment (mariano-aguero) [reply app/src/components/common/button_connect_wallet/index.tsx]: Done

File: app/src/components/common/modal_connect_wallet/index.tsx (hunk 1 above)
Comment (mariano-aguero) [reply app/src/components/common/modal_connect_wallet/index.tsx]: Done

File: app/src/components/common/modal_connect_wallet/index.tsx (hunk 2 above)
Comment (mariano-aguero) [reply app/src/components/common/modal_connect_wallet/index.tsx]: Done

**Pull request merged**